File: All Codes/URI_1012.java
import java.util.Scanner;
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class URI_1012 {
 
    public static void main(String[] args) {
 
        try (Scanner input = new Scanner(System.in)) {
            double a, b, c;

            a = input.nextDouble();
            b = input.nextDouble();
            c = input.nextDouble();

            System.out.printf("TRIANGULO: %.3f\n", (a * c) / 2);
            System.out.printf("CIRCULO: %.3f\n", c * c * 3.14159);
            System.out.printf("TRAPEZIO: %.3f\n", ((a + b) / 2) * c);
            System.out.printf("QUADRADO: %.3f\n", b * b);
            System.out.printf("RETANGULO: %.3f\n", a * b);

        }
 
    }
 
}

File: All Codes/URI_1012.py
angle = [float(x) for x in input().split()]

TRIANGULO =  angle[0] * angle[2]  * 0.5
CIRCULO   =  angle[2] * angle[2]  * 3.14159
TRAPEZIO  =( angle[0] + angle[1] )* angle[2] * 0.5
QUADRADO  =  angle[1] * angle[1]
RETANGULO =  angle[0] * angle[1]

print(f"TRIANGULO: {TRIANGULO:.3f}")
print(f"CIRCULO: {CIRCULO:.3f}")
print(f"TRAPEZIO: {TRAPEZIO:.3f}")
print(f"QUADRADO: {QUADRADO:.3f}")
print(f"RETANGULO: {RETANGULO:.3f}")